scripts/globals/mobskills/Slaverous_Gale.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Slaverous Gale
+--
+--  Description: Deals earth damage that inflicts Plague and Slow effects on targets in front of the caster
+--  Type: Magical (Earth)
+--
+--
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target,mob,skill)
+	local typeEffectOne = EFFECT_PLAGUE;
+	local typeEffectTwo = EFFECT_SLOW;
+	MobStatusEffectMove(mob,target,typeEffectOne,1,3,60);
+	MobStatusEffectMove(mob,target,typeEffectTwo,1,3,60);
+	local dmgmod = 1; 
+    local info = MobMagicalMove(mob, target, skill, mob:getWeaponDmg() * math.random(4,6),ELE_EARTH,dmgmod,TP_NO_EFFECT);
```

Comment:
Ok will do.  MOST of the time FFXIclopedia is where it seems most reliable. This one, http://ffxiclopedia.wikia.com/wiki/Gastric_Bomb.  It had BLU Final Sting HP to 1, but I can't find one for bees that says it, so I can put that thing back where it came from!  My indents look ok on Notepad++, I'm not sure what happened there.  Cyclonic Turmoil shouldn't have gotten in here in this round, oops.  And I'm not sure how those end lines even get there.